marketplace_builder/views/pages/release-notes/18-FEBRUARY-2019.liquid

```diff
@@ -0,0 +1,28 @@
+---
+converter: markdown
+metadata:
+  title: February 18, 2019  &ndash;  Release Notes
+  date: February 18, 2019
+  description: Properties for all users, Liquid in page metadata
+  hide_sidebar: true
+  exclude_contributors: true
+slug: release-notes/18-FEBRUARY-2019
+searchable: true
+---
+
+{{ page.metadata.description }}
+
+## What's new
+
+* **Properties for all users**: You can now set properties directly in the user object, no need to go through the user profile. You can define properties all users should have via the marketplace_builder/user.yml file.
+* **Liquid in page metadata**: Page metadata now accepts Liquid and it will evaluate it for each page render. Same for layout. Of course, it’s lazy evaluation - happens only when you use it.
```

Comment:
```suggestion
* **Liquid in page metadata**: Page metadata now accepts Liquid and it will evaluate it for each page render. Same for layout.
```

:)

Reply:
sigh, ok :)